lib/plugins/Tests/Tests.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
+class Tests {
+  constructor() {
+    this.commands = {
+      test: {
+        commands: {
+          integration: {
+            usage: 'Command for integration testing.',
+            lifeCycleEvents: [
+              'createFile',
+            ],
+          },
+        },
+      },
+    };
+
+    this.hooks = {
+      'test:integration:createFile': this.createFile,
+    };
+  }
+
+  createFile() {
```

Comment:
Imho it would be better that instead of writing a file (which is always tricky because it requires cleanup) this plugin could simply log the current command name and all parameters that are given. Then in the integration test we could simply read the log output the command is producing and making sure that the log output conforms to what we want it to be. This way we check that it can properly execute the whole stack of the command, parameters are correctly handed down, ...